core/src/MsalIdHelper.cs

```diff
@@ -66,38 +66,32 @@ internal static class MsalIdParameter
     /// </summary>
     internal static class MsalIdHelper
     {
-        private static CorePlatformInformationBase corePlatformInformationBaseInstance;
```

Comment:
Minor rewrite - just CorePlatformInformationBase.Instance - it's easier to read.